dev_setup.sh

```diff
@@ -115,6 +115,34 @@ function get_YN() {
     done
 }
 
+function os_is() {
+    [[ $(grep "^ID=" /etc/os-release | awk -F'=' '/^ID/ {print $2}' | sed 's/\"//g') == $1 ]]
+}
+
+function os_is_like() {
+    [[ $(grep "^ID_LIKE=" /etc/os-release | awk -F'=' '/^ID_LIKE/ {print $2}' | sed 's/\"//g') == $1 ]]
+}
+
+if os_is_like debian || os_is debian || os_is_like ubuntu || os_is ubuntu || os_is linuxmint; then
+    if dpkg -V lib-jack-2-0 ; then
```

Comment:
We might want to use `dpkg -V lib-jack-2-0 > /dev/null 2>&1` to silence the output from the check.